sema/commons/store/store.py

```
@@ -414,37 +423,37 @@ def _update_registry_lastmod(
+            adm_graph.remove(pattern)  # type: ignore
         else:
             response = [
-                str(sub) for (sub, pred, obj) in adm_graph.triples(pattern)
+                str(sub) for (sub, pred, obj) in adm_graph.triples(pattern)  # type: ignore # noqa
             ]
 
         # insert the new data if provided
         if graph_subject is not None and lastmod is not None:
             triple = tuple(
                 (graph_subject, SCHEMA_DATEMODIFIED, Literal(lastmod))
             )
```

Ruff

Unnecessary `tuple` literal passed to `tuple()` (remove the outer call to `tuple()`)

Remove the outer call to `tuple()`

(C409)